charts/base-cluster/templates/monitoring/kube-prometheus-stack/_alertmanager-config.yaml

```diff
@@ -1,10 +1,15 @@
-{{- define "base-cluster.prometheus-stack.alertmanager.config" -}}
-{{- $enabled := false -}}
-{{- range $_, $receiver := .Values.monitoring.prometheus.alertmanager.receivers -}}
-  {{- $enabled = or $enabled $receiver.enabled -}}
+{{- define "base-cluster.prometheus-stack.alertmanager.enabled" -}}
+  {{- $enabled := false -}}
+  {{- range $_, $receiver := .Values.monitoring.prometheus.alertmanager.receivers -}}
+    {{- $enabled = or $enabled $receiver.enabled -}}
+  {{- end -}}
+  {{- $enabled | ternary true "" }}
```

Comment:
Why `true` or empty and not `true` or `false?`

Reply:
Because helm template works with truthy values:

`{{ if "false" }} this is rendered {{ end }}`
`{{ if "" }} this is not rendered {{ end }}`

Reply:
Alright :+1: